i18n/de.json

```diff
@@ -1505,6 +1505,7 @@
   "app.channel.post_update_channel_purpose_message.removed": "{username} hat den Kanalzweck entfernt (war: {old})",
   "app.channel.post_update_channel_purpose_message.updated_from": "{username} hat den Kanalzweck aktualisiert von: {old} auf: {new}",
   "app.channel.post_update_channel_purpose_message.updated_to": "{username} hat den Kanalzweck geändert zu: {new}",
+  "applications": "Applications",
```

Comment:
We only add translations to the `en.json` file, the rest of the translations are handled by our translation process, please revert all the changes to `i18n/xx.json` files except for the `i18n/en.json` file.

Reply:
done